pyvision3/montage.py

```diff
@@ -482,9 +488,8 @@ def __next__(self):
             image_list.append(self.imgs[k])
 
         # create an image montage from the current video frames and advance the frame counter
-        im = ImageMontage(image_list, self.layout, self.vid_size, gutter=2, by_row=True, labels=keys)
+        im = ImageMontage(image_list, self.layout, self.vid_size, gutter=2, by_row=True, labels=keys, alpha=self.alpha)
```

Comment:
Nitpick: 120 chars in this line.  Consider breaking this call to the `ImageMontage` constructor into multiple lines.

Reply:
Yes, for sure. This repo predates "black", and I haven't spent the time to setup github actions to help keep the code clean & tidy. So I'll run black on this file, and then run black on all files in the repo as a separate PR that is a "reset" on the formatting.